sanket/javascript/assignment10(day28)/sanketA10_Q5.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>count number of words in string.</title>
+  </head>
+  <body>
+    <textarea id="t" cols="40" rows="10"></textarea>
+    <button onclick="trimString()">trim</button>
+    <script>
+      function trimString() {
+        let string = document.getElementById("t").value;
+        let newString = string.replace(/\s+/g, " ");
+        document.write("number of words: " + newString.split(" ").length);
```

Comment:
instead of document.write use a p tag and set the innerhtml for it.
this is an advice, u can merge this code